crates/node/src/chain_spec.rs

```diff
@@ -153,12 +154,13 @@ pub fn local_testnet_config(base_path: BasePath, chain_id: &str) -> Result<Chain
 }
 
 fn load_genesis(data_path: PathBuf) -> GenesisLoader {
+
```

Comment:
What with the empty line at the beginning of the function body? But removing it before the return?